bust-a-move-framework/src/main/java/bustamove/App.java

```diff
@@ -24,88 +26,115 @@
  */
 
 public class App extends StateBasedGame {
-    /**
-     * Dimensions of the game window.
-     */
-    private static final int GAME_WIDTH = 640;
-    private static final int GAME_HEIGHT = 580;
-    /**
-     * Values for fps of the game.
-     */
-    private static final int MAX_FPS = 120;
+	/**
+	 * Dimensions of the game window.
+	 */
+	public static final int GAME_WIDTH = 640;
+	public static final int GAME_WIDTH_MULTIPLAYER = 1040;
+	public static final int GAME_HEIGHT = 580;
```

Comment:
I fixed most of the violations but there are still 10 left. The Travis will fail because of this. Maybe you can take a look at the errors.